Tests/iaas/flavor-naming/flavor_names.py

```diff
@@ -226,13 +244,29 @@ class GPU:
     vram = OptIntAttr("#.V:GiB VRAM")
     vramperf = TblAttr("Bandwidth", {"": "Std BW {<~1GiB/s)", "h": "High BW", "hh": "Very High BW"})
 
+    def __init__(self, gputype="g", brand="N", gen='', cu=None, perf='', vram=None, vramperf=''):
```

Comment:
I'd need to double check that we are not defaulting to having a virtualized NVidia GPU this way...

Reply:
The constructor is only used in one place, namely shorten, and there gputype and brand are propagated.